reference/orders.v2.oas2.yml

```diff
@@ -412,10 +431,54 @@ paths:
                           value: '180'
                         - id: 230
                           value: '192'
+              Shipping consignment:
```

Comment:
Missing `customer_id` in this?

Reply:
it's optional, defaults to guest customer id of `0`